pkg/registry/generic/rest/streamer.go

```diff
@@ -56,6 +59,16 @@ func (s *LocationStreamer) InputStream(apiVersion, acceptHeader string) (stream
 	if err != nil {
 		return
 	}
+
+	if resp.StatusCode == http.StatusInternalServerError {
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
I'm not sure this code is what should be doing this either - some streamers may want to handle the response themselves for structured error types